### ISSUE TITLE
Support with file structure

### ISSUE THREAD

**Issue by thomasgregg** [nechry/elco-remocon-net-appdaemon#19]
Hi,

Thanks so much for developing this appdaemon. I recently used your red node implementation but now setup a new HA instance and would like to give this one a try.

Where I am really stuck / confused is about where and which files and folders need to be exactly manually copied to. Would it be possible to send me a directory listing that exactly shows all the files and folders and where they have to go?

Thanks so much.

**Comment by nechry** [comment]
Sorry Thomas for the late reply...
Check out the [HA Community Forum](https://community.home-assistant.io/t/elco-remocon-heating-integration/49698/116?u=nechry): I've already posted the exact screenshot you're looking for.
Let me know if you need more details

**Comment by nechry** [comment]
I just added an item to the readme for clarity, as it looks like HACS/or add-ons aren't changing anytime soon.

**Comment by wurstconsal** [comment]
Hi, I am struggling with a similar issue but in my case, even if I reproduce the folder and file structure as in the screenshot, I still get an error when starting Appdaemon.
For clarity, I installed the integration from HACS and I moved the addon from /config/appdaemon/apps to /addon_configs/<guid>_appdaemon/apps:

<img width="1041" height="296" alt="Image" src="https://github.com/user-attachments/assets/aeec2b35-42c5-47cf-9318-27a06984ae0b" />

Still, after trying many things and variations, I still get errors when launching AppDemon, it seems it is still searching for the addon in /config/ but even if I put it there again, I still get errors

Below the log and I attached my apps.yaml file. Any idea of what am I doing wrong?

[01:08:19] INFO: Starting AppDaemon...
s6-rc: info: service legacy-services successfully started
2025-12-24 01:08:27.533262 INFO AppDaemon: ------------------------------------------------------------
2025-12-24 01:08:27.533518 INFO AppDaemon: AppDaemon Version 4.5.12 starting
2025-12-24 01:08:27.533686 INFO AppDaemon: ------------------------------------------------------------
2025-12-24 01:08:27.533836 INFO AppDaemon: Python version is 3.12.12
2025-12-24 01:08:27.534046 INFO AppDaemon: Configuration read from: /config/appdaemon.yaml
2025-12-24 01:08:27.535052 INFO AppDaemon: Using /config/apps as app_dir
2025-12-24 01:08:27.537060 INFO AppDaemon: Loading built-in plugin 'HASS' using 'HassPlugin' from 'appdaemon.plugins.hass.hassplugin'
2025-12-24 01:08:27.537477 INFO HASS: HASS Plugin initialization complete
2025-12-24 01:08:27.538017 INFO AppDaemon: Initializing HTTP
2025-12-24 01:08:27.538686 INFO AppDaemon: Using 'ws' for event stream
2025-12-24 01:08:27.543499 INFO AppDaemon: Starting API
2025-12-24 01:08:27.546453 INFO AppDaemon: Starting Admin Interface
2025-12-24 01:08:27.546887 INFO AppDaemon: Starting Dashboards
2025-12-24 01:08:27.553189 INFO AppDaemon: Scheduler running in realtime
2025-12-24 01:08:27.555417 INFO AppDaemon: Starting apps with 1 worker threads. Apps will all be assigned threads and pinned to them.
2025-12-24 01:08:27.564156 INFO AppDaemon: Running on port 5050
2025-12-24 01:08:27.565099 INFO AppDaemon: Waiting for plugins to be ready
2025-12-24 01:08:27.570281 INFO HASS: Connected to Home Assistant 2025.12.4 with aiohttp websocket
2025-12-24 01:08:27.572476 INFO HASS: Authenticated to Home Assistant 2025.12.4
2025-12-24 01:08:27.576338 INFO HASS: Waiting for Home Assistant to start
2025-12-24 01:08:27.607421 INFO AppDaemon: All plugins ready
2025-12-24 01:08:27.621174 INFO HASS: Completed initialization in 70.526ms
2025-12-24 01:08:27.636433 ERROR Error: =====  Error importing 'elco_remocon'  ====================================
2025-12-24 01:08:27.638310 ERROR Error: FailedImport: Failed to import 'elco_remocon'
2025-12-24 01:08:27.638717 ERROR Error:   Import paths:
2025-12-24 01:08:27.639002 ERROR Error:     /config/apps
2025-12-24 01:08:27.639380 ERROR Error:     /config/apps/elco-remocon-net-appdaemon/apps/elco-remocon-net-appdaemon
2025-12-24 01:08:27.642219 ERROR Error:   ModuleNotFoundError: No module named 'elco_remocon'

[apps.yaml](https://github.com/user-attachments/files/24319945/apps.yaml)
